spec/haskell/src/SEL4/Object/Structures/AARCH64.hs

```diff
@@ -70,6 +70,11 @@ atcbContextSet uc atcb = atcb { atcbContext = uc }
 atcbContextGet :: ArchTCB -> UserContext
 atcbContextGet = atcbContext
 
+data ArchTcbFlag = FpuDisabled
+
+archTcbFlagToWord :: ArchTcbFlag -> Word
+archTcbFlagToWord (FpuDisabled) = 0x1
```

Comment:
No desire for `bit`-equivalent here?

Reply:
Good point, I missed that. Thinking about it now though, the equivalent in C produces values with the final mask, so at some point we'll need to do the conversion between that and the `bit` form. Do we think that that's simple enough that it can be done wherever, or would it be better do it in `Refine` instead of `CRefine`?

Reply:
No idea. It should simplify out, just look more sensible in Haskell, especially if we get future flags.

Reply:
I agree that it should simplify out, I've made this change now.